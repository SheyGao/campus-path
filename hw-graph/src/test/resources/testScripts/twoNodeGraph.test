# Test if two nodes can be added

# Create a Graph
CreateGraph graph2

# Add a node to the graph
AddNode graph2 a
AddNode graph2 b

# Print the nodes in the graph, the outgoing edges from a, the outgoing edges from b
ListNodes graph2
ListChildren graph2 a
ListChildren graph2 b