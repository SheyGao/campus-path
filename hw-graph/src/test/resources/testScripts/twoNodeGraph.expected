# Test if two nodes can be added

# Create a Graph
created graph graph2

# Add a node to the graph
added node a to graph2
added node b to graph2

# Print the nodes in the graph, the outgoing edges from a, the outgoing edges from b
graph2 contains: a b
the children of a in graph2 are:
the children of b in graph2 are: